### PR TITLE
Use sbt-conductr-sandbox

### PR DESCRIPTION
**Pull request by markusjura**
Make use of `sbt-conductr-sandbox`. Also upgrades the the project to Play 2.4.2 and the latest libs.

### CODE REVIEW

**Comment by huntc** [comment]
@markusjura Is this still a WIP?

**Comment by markusjura** [comment]
Yes, because the `build.sbt` is not working correctly. What I'd like to achieve is that:

**Requirements**
1. `bundle:dist` in the root project should create the bundles for the 2 sub projects
2. All `conduct` commands incl. `conduct info` and `conduct load` should work in the root project
3. All `sandbox` commands should work in the root project
4. Optional: `conduct load <HIT TAB KEY>`  should rotate through the bundles files in all sub projects (enhancement in `sbt-conductr-sandbox` necessary)

**Current state**
- Requirement 1. and 3. is working
- `conduct info` throws this error message if executed in root project:
  
  ```
  [trace] Stack trace suppressed: run last ferryService/*:conduct for the full output.
  [error] (ferryService/*:conduct) akka.actor.InvalidActorNameException: actor name [screen] is not  unique!
  [error] Total time: 2 s, completed Aug 31, 2015 9:00:55 PM
  ```
- Executing `conduct load` in root project results into this error (it only works in the subprojects):
  
  ```
  [trace] Stack trace suppressed: run last charonShopApp/*:conduct for the full output.
  ```
- `conduct load <HIT TAB KEY>` only resolves the filename of the first bundle.

**Comment by markusjura** [comment]
Another issue is that the application is shutting down as soon as it is receiving a request.

**Comment by markusjura** [comment]
The issue that the application is shutting down has been resolved.

The **Requirements** will be tackled by changing the `conduct` command in `sbt-conductr` to global setting. The corresponding issue is: https://github.com/sbt/sbt-conductr/issues/99

@edwardcallahan: Please review and merge. Also it would be nice if you could transfer this repo to typesafehub.

File: README.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ Build both application bundles using bundle:dist
 Load and start both bundles to ConductR. You should have two services available:
 
 ```bash
-SERVICE              BUNDLE ID  BUNDLE NAME  STATUS
-http://:9000/charon  386078b    charon-shop  Running
-http://:9666/ferry   cc08f55    ferry-boat   Running
+ID       NAME         #REP  #STR  #RUN
+5ba0bd0  ferry-boat      3     0     3
+52b15ab  charon-shop     3     0     3
```
Comment (huntc) [inline README.md]: @markusjura Ed is using the `conduct services` command here.

File: play-front/app/actors/Delivery.scala [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+
+  def props: Props = Props(new Delivery)
 }
 
 class Delivery extends Actor with ActorLogging {
   import Delivery._
   implicit val timeout = Timeout(5.seconds)
   implicit val executionContext = context.dispatcher
-  val ferry = LocationService.getLookupUrl("/ferry", "http://127.0.0.1:9666")
+  val ferry = LocationService.getLookupUrl("/ferry", URL("http://127.0.0.1:9666/ferry"))
```
Comment (huntc) [inline play-front/app/actors/Delivery.scala]: Should we really bind to 127.0.0.1 here? I think that this should instead read:

```scala
  val ferry = LocationService.getLookupUrl("/ferry", URL("http://:9666/ferry"))
```

Comment (huntc) [inline play-front/app/actors/Delivery.scala]: ignore

File: README.md (hunk 1 above)
Comment (markusjura) [inline README.md]: Ok, reverting this change.

**Comment by edwardcallahan** [comment]
Forked into Typesafehub. Thank you @markusjura

**Pull request merged**